README.md

```diff
@@ -132,7 +135,9 @@ This command has two required flags:
 
 `--private-key` - loadbalancers wallet private key (string representation of hex value prefixed with 0x), used for sending founds on payout
 
-`--payout-reward` - defined total reward amount that will be distributed on the payout (amount in Planck)
+`--payout-reward` - defined total reward amount that will be distributed on the payout (amount in Planck). If omitted entire balance of lb wallet will be used as total reward. If this options is selected than `--lb-payout-fee-address` must be set
```

Comment:
Wait no, if --payout-reward is omitted, --lb-payout-fee-addressmust be set

Reply:
That's what I meant (by **option** I meant option of sending entire balance a.k.a omitting `--payout-reward` flag) but now I see how this is very confusing wording. Fill fix